drivers/watchdog/wdt_iwdg_stm32.c

```diff
@@ -39,7 +43,7 @@
 
 /*
  * Status register needs 5 LSI clock cycles divided by prescaler to be updated.
- * With highest prescaler (256) and considering clock variation, we will wait
+ * With highest prescaler (256 or 1024 for STM32U5) and considering clock variation, we will wait
```

Comment:
You will need to add a line break, otherwise it will not pass CI (max. 80 characters per line are allowed).